core/src/banking_stage/consumer.rs

```diff
@@ -2318,9 +2319,14 @@ mod tests {
                     1,
                     bank.last_blockhash(),
                 ));
+            let feature_set: Arc<FeatureSet> = bank.feature_set.clone();
+            let allow_self_conflicting_txns =
+                feature_set.is_active(&feature_set::allow_self_conflicting_entries::id());
```

Comment:
no need to clone feature-set. just get the bool
```suggestion
            let allow_self_conflicting_txns =
                bank.feature_set.is_active(&feature_set::allow_self_conflicting_entries::id());
```